TestFoundation/TestNSISO8601DateFormatter.swift

```diff
@@ -7,6 +7,8 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
```

Comment:
nit: We can call this file `TestISO8601DateFormatter.swift` if we are okay with dropping the NS prefix from the source file name as well